index.test.ts

```diff
@@ -244,11 +245,47 @@ describe("Replicate client", () => {
         value: Buffer.from("hello world"),
         expected: "data:application/octet-stream;base64,aGVsbG8gd29ybGQ=",
       },
-    ])(
+    ];
+
+    test.each(fileTestCases)(
+      "converts a $type input into a Replicate file URL",
+      async ({ value: data, expected }) => {
+        nock(BASE_URL)
+          .post("/files")
+          .reply(201, {
+            urls: {
+              get: "https://replicate.com/api/files/123",
+            },
+          })
+          .post("/predictions")
+          .reply(201, (uri: string, body: Record<string, any>) => {
+            return body;
```

Comment:
Probably worth doing some verification on the `FormData` object here, this might have caught the `fileName` bug.